pkger/models.go

```diff
@@ -933,6 +944,19 @@ func (c chart) validBaseProps() []failure {
 	return fails
 }
 
+func (c chart) validNote() []failure {
```

Comment:
nit: you can do this inline, don't think anything else will be reusing this

Reply:
also, is a `""` a valid note for markdown? not sure what our validation is for markdowns 🤷‍♂